resources/integration_test/testdata/integration.yml

```diff
@@ -0,0 +1,19 @@
+dataset:
+  - db: integration_test
+    collection: class
+    docs:
+      - {studentid: 10329,  name: "John", enrolled: true}
```

Comment:
I think this is less of a worry because this is testing the driver and not mongosql. As long as it's some kind of number I think it's fine (i.e., no worries about smallest type).

Reply:
The numeric type for `studentid` that the `sqlGenerateSchema` command returns is `int`.
`testid` is also `int`.  `score` is `double`.